src/transformers/models/gpt2/configuration_gpt2.py

```diff
@@ -61,6 +61,8 @@ class GPT2Config(PretrainedConfig):
             Dimensionality of the inner feed-forward layers. :obj:`None` will set it to 4 times n_embd
         activation_function (:obj:`str`, `optional`, defaults to :obj:`"gelu"`):
             Activation function, to be selected in the list :obj:`["relu", "silu", "gelu", "tanh", "gelu_new"]`.
+        attn_layer_scale (:obj:`bool`, `optional`, default to :obj:`False`):
+            Argument used to turn on the attention scale by 1 / layer_num for each block
```

Comment:
@lorr1 - can we add like a [Mistral-GPT] tag here?

Reply:
Should we? Like someone can use this without loading a mistral model? Are there other examples of like "flag for XXX model" in the code? Happy to do it if there is, but I'm not sure what's standard.

Reply:
There aren't, because most cases (GPT-Neo) define brand new models. For the sake of backwards compatibility, and clear signaling that these are necessary flags to set when loading Mistral models (also see Slack about the `from_pretrained()` question), I think we should have them?

Also - when we put in the final PR, the HF folks can tell us to remove.